### DISCUSSION TITLE
Would Sandboxie contain kernel-based monitoring software?

### DISCUSSION THREAD

**Discussion by qualityrice** [sandboxie-plus/Sandboxie#3662] [General]
Some video games use kernel-based anticheat software to function, meaning they monitor the system from the OS kernel even when the game isn't running, Riot Vanguard for example. The purpose of this question is not to to try and bypass anticheat software while a game is running, but to to make sure the anticheat software isn't running when the game is closed, for privacy reasons.

Could Sandboxie do this? Containing something as complex as kernel-based software? Would that kernel-based monitoring be disabled when the program is terminated with Sandboxie?

**Comment by bastik-1001** [comment]
It's not possible to isolate kernel drivers, as [David stated here](https://forum.xanasoft.com/threads/sandboxie-and-kernel-drivers.531/).

And something that is installed on the system with kernel privileges is most likely able to run in the background, even if the game itself is installed in a sandbox (or started there).

Anti-cheat running while no game is running is something people might not like. At least, it can be disabled by the user, although playing another game requires rebooting the system as the anti-cheat software is loaded that early.
